Apollo.xcodeproj/project.pbxproj

```diff
@@ -1141,7 +1143,8 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
```

Comment:
Should this _not_ be `/bin/sh`? 🤔

Reply:
that's the default that apple has been using pre-Catalina. I'll have to see what they're doing for scripts on Catalina, but for most intents and purposes this should work fine.

Reply:
Yeah, leaving this is fine, and I wouldn't be surprised if it remains the same in Catalina since I think their intention is just to change the user's default shell.  Either way, the presence of the shebang in the scripts will ensure that the OS runs the desired interpreter.  In other words, this can be surely resolved. 😉